src/layouts/AppLayout/IconSidebar/IconSidebar.tsx

```diff
@@ -22,20 +22,13 @@ const Icons = {
 	goals: <Goals />,
 	income: <Income />,
 	payment: <Payment />,
-	"irregular expenses founds": (
-		<IrregularExpensesFunds />
-	),
+	"irregular expenses founds": <IrregularExpensesFunds />,
```

Comment:
dlaczego taki klucz zamiast cammelCase?

Reply:
mam taki obiekt w routes np.
{
		path: Paths.IRREGULAR_EXPENSES,
		element: <IrregularExpensesFund />,
		title: "irregular expenses founds",
		isProtected: true,
		layout: "app",
	},
Tutaj wykorzystuje title z routes do wybrania  icon `Icons[title]  ` 
Zastanawiałąm sie czy nie dodać właściwości icon do powyższego obiektu ale stwierdziałm, ze wykorzystam "titlle" 
Sugerujesz by dodać?

Reply:
Tak. Albo zrobić je bardziej niezależne albo tak jak piszesz dodać do tego configu pole icon. Teraz to powiązanie po title jest mało responsywne i nie odporne na zmianę czy tłumaczenia na inne języki.

Reply:
Zmienione